runtime/tools/spec-builder/README.md

```diff
@@ -0,0 +1,107 @@
+# Runtime Spec-Builder Tool
+
+The tool is intended to assist AssemblyScript Runtime developers into generating their Genesis Chain spec files. It is not published as a npm library, thus you will need to install and build it locally when you are using it.
+
+### Usage
+1. Install the tool by executing the following command:
+```
+yarn install
+```
+2. Build the tool by executing the following command:
```

Comment:
maybe more correct wording would be "Build the wasm module (or Assemblyscript module)", since the _tool_ is mostly JS and it doesn't need building?